### PR TITLE
Update README.es

### LINKED ISSUE #6240: Add translations for "No Login Required" in corresponding Readme.md files
### Checklist

- [x] I am using the latest version - x.xx.x 
- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Task
Translations for feature "No Login Required" needs to be added in Readme files of corresponding languages.
Related [PR#5994](https://github.com/TeamNewPipe/NewPipe/pull/5994)

### PR DESCRIPTION
**Pull request by CloverBadland**
#### What is it?
- [ ] Bugfix (user facing)
- [ ] Feature (user facing)
- [ ] Codebase improvement (dev facing)
- [x] Meta improvement to the project (dev facing)

#### Description of the changes in your PR

- Spelling fixes
- Changed links
- Added information for more details
- Added a reference link to the blog in the privacy policy section.

These are the main changes

#### Fixes the following issue(s)

- Bad translations
- Spelling errors
- NewPipe version image doesn't match the latest version
- Fixes #6240 (Only in the Spanish README)

#### Due diligence
- [x] I read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by mhmdanas — CHANGES_REQUESTED** [review]
Can you please fix the merge conflicts?

File: README.es.md [hunk 1]
```diff
@@ -1,30 +1,30 @@
...
+<p align="center"><a href="https://f-droid.org/packages/org.schabi.newpipe/"><img src="https://fdroid.gitlab.io/artwork/badge/get-it-on-es.svg"></a></p>
 
 <p align="center">
-<a href="https://github.com/TeamNewPipe/NewPipe/releases" alt="GitHub release"><img src="https://img.shields.io/badge/Lanzamiento-v0.20.11-blue.svg" ></a>
+<a href="https://github.com/TeamNewPipe/NewPipe/releases" alt="GitHub release"><img src="https://img.shields.io/github/release/TeamNewPipe/NewPipe.svg" ></a>
 <a href="https://www.gnu.org/licenses/gpl-3.0" alt="License: GPLv3"><img src="https://img.shields.io/badge/Licencia-GPL%20v3-blue.svg"></a>
 <a href="https://github.com/TeamNewPipe/NewPipe/actions" alt="Build Status"><img src="https://github.com/TeamNewPipe/NewPipe/workflows/CI/badge.svg?branch=dev&event=push"></a>
-<a href="https://hosted.weblate.org/engage/newpipe/es/" alt="Estado de la traducción"><img src="https://hosted.weblate.org/widgets/newpipe/es/svg-badge.svg"></a>
+<a href="https://hosted.weblate.org/engage/newpipe/es/" alt="Translation Status"><img src="https://hosted.weblate.org/widgets/newpipe/es/svg-badge.svg"></a>
```
Comment (mhmdanas) [inline README.es.md:11]: Why change this back to English?

File: README.es.md [hunk 2]
```diff
@@ -40,77 +40,79 @@
...
-Si tiene ideas, traducciónes, cambios de diseño, limpieza de código, o cambios grandes de código, su ayuda es siempre bienvenida.
-Cuanto más realizamos, mejor se pone la aplicación!
+
+Si tiene ideas, traducciones, cambios de diseño, limpieza de código o cambios grandes de código, su ayuda es siempre bienvenida. ¡Cuanto más hagamos, NewPipe será mucho mejor!
 
 Si quiere involucrarse, fíjese en nuestras [notas de contribución](.github/CONTRIBUTING.md).
 
 <a href="https://hosted.weblate.org/engage/newpipe/es/">
-<img src="https://hosted.weblate.org/widgets/newpipe/es/287x66-grey.png" alt="Estado de la traducción" />
+<img src="https://hosted.weblate.org/widgets/newpipe/es/287x66-grey.png" alt="Translation status" />
```
Comment (mhmdanas) [inline README.es.md:111]: Same thing; why change this back to Engllsh?

File: README.es.md (hunk 1 above)
Comment (CloverBadland) [reply README.es.md:11]: To keep the same translation of all the alt attributes in the document (all are in English except this one and one other)

File: README.es.md (hunk 2 above)
Comment (CloverBadland) [reply README.es.md:111]: To keep the same translation of all the alt attributes in the document (all are in English except this one and one other)

I could change all this to Spanish, if necessary.

Comment (mhmdanas) [reply README.es.md:111]: There's no point to `alt` attributes if the reader can't understand them, so yes, please translate them all to Spanish.

**Review by mhmdanas — APPROVED** [review]
LGTM, thanks!

**Pull request merged**